src/uikit.cs

```diff
@@ -11254,8 +11553,12 @@ interface UISearchDisplayDelegate {
 	[BaseType (typeof (NSObject))]
 	partial interface UISearchResultsUpdating {
 		[Abstract]
-	    [Export ("updateSearchResultsForSearchController:")]
-	    void UpdateSearchResultsForSearchController (UISearchController searchController);
+		[Export ("updateSearchResultsForSearchController:")]
+		void UpdateSearchResultsForSearchController (UISearchController searchController);
```

Comment:
Do we need 'ForSearchController'?

Reply:
We can't change it, it would be a breaking change :) this is not new, looks like I fixed indentation